### PR TITLE
Config files

### PR DESCRIPTION
**Pull request by dennisklein**
Followup on #723

### CODE REVIEW

**Comment by dennisklein** [comment]
Now the CI builds are running. So, if one closes a PR, then updates the forked branch and reopens the PR, github recognizes the new commit, but does not update the internal PR head ^^

**Comment by dennisklein** [comment]
@miharenthas Something is not yet right with the switch statement: https://cdash.gsi.de/testDetails.php?test=4050506&build=170622

```
line 49: syntax error near unexpected token `-a'
line 49: `		-a | --append )'
```

@fuhlig1 @MohammadAlTurany What do you think about the latest resolution (once it is working)?

File: cmake/scripts/config.sh.in [hunk 1]
```diff
@@ -4,48 +4,62 @@ export Linux_Flavour_="@_linux_flavour@"
...
+		export G4ENSDFSTATEDATA="@G4ENSDFSTATEDATA@"
+	fi
+	export ROOT_LIBRARY_DIR="@ROOT_LIBRARY_DIR@"
+	export ROOT_LIBRARIES="@ROOT_LIBRARIES@"
+	export ROOT_INCLUDE_DIR="@ROOT_INCLUDE_DIR@"
+	export ROOT_INCLUDE_PATH="@ROOT_INCLUDE_PATH@"
+	export VMCWORKDIR="@VMCWORKDIR@"
+	export FAIRLIBDIR="@FAIRLIBDIR@"
+	export PYTHONPATH="@MY_PYTHONPATH@"
+	case $1
```
Comment (dennisklein) [inline cmake/scripts/config.sh.in]: I would guess this needs to be `case "$1" in` ?

Comment (miharenthas) [reply cmake/scripts/config.sh.in]: Hi, no, something else is missing: the construct is ´case $var in´: I forgot the in. Weird: I did some tests...

Anyway I'm pushing now again with the correction.

**Comment by fuhlig1** [comment]
Closed with commit 060cf2bd0245ce444d15069989265ed95c1acb44. I squashed the three commits and adapted the commit message.

**Pull request closed without merging**